fontique/src/font.rs

```diff
@@ -230,6 +257,30 @@ const ITALIC_AXIS: u8 = 0x08;
 const OPTICAL_SIZE_AXIS: u8 = 0x10;
 
 /// An axis of variation for a variable font.
+///
+/// Instances of this can be obtained from [`FontInfo::axes`].
+///
+/// These give the [`Tag`] and range of valid values for a given font
+/// variation. In `parley`, these values are used to create a
+/// `FontVariation`.
+///
+/// OpenType defines some common axes:
+///
+/// * [Italic](https://fonts.google.com/knowledge/glossary/italic_axis) or `ital`
+/// * [Optical Size](https://fonts.google.com/knowledge/glossary/optical_size_axis) or `opsz`
+/// * [Slant](https://fonts.google.com/knowledge/glossary/slant_axis) or `slnt`
+/// * [Weight](https://fonts.google.com/knowledge/glossary/weight_axis) or `wght`
+/// * [Width](https://fonts.google.com/knowledge/glossary/width_axis) or `wdth`
+///
+/// # See also
+///
+/// * [`FontInfo::axes()`]
+/// * [`FontInfo::has_italic_axis()`]
+/// * [`FontInfo::has_optical_size_axis()`]
+/// * [`FontInfo::has_slant_axis()`]
+/// * [`FontInfo::has_weight_axis()`]
+/// * [`FontInfo::has_width_axis()`]
```

Comment:
I think we should instead link directly to these at the relevant item in the list above.
With possibly a sentence like:

> FontInfo allows cheaply querying for the presence of these axes using the linked methods.

Reply:
I've just removed them instead as I didn't like the other options. I think that's a net negative that this info is gone.